feature/login.feature

```diff
@@ -1,52 +1,51 @@
+    Then o usuário de CPF "00000000013" é encaminhado para a pagina de "administrador"
 
 Scenario: Login incorreto
     Given o usuário está na página de "login"
-    And existe um usuário cadastrado
-    When eu tento acessar o aplicatico com o cpf "11111111111"
-    And o campo da senha é preenchido por "123456"
-    Then eu recebo uma mensagem  "Credenciais incorretas"
-    And eu continuo na página de login
+    And existe um usuário de CPF "12989087064" cadastrado no sistema
+    When o usuário tenta fazer login com o email "usererror@gmail.com" e senha "1234567a"
+    Then o usuário recebe uma mensagem  "Credenciais incorretas."
+    And o usuário continua na página de "login"
 
+Scenario: Esqueceu a senha
+    Given o usuário está na página de "login"
+    When o usuário seleciona a opção de esqueceu a senha
+    Then o usuário é encaminhado para a pagina de "esqueci a senha" 
 
-Scenario: Esqueceu a senha e preenche o campo de recuperar com o código certo
-    Given o usuário "acruchi" está na página de "login"
-    When o usuário "acruchi" seleciona a opção de Esqueceu a senha
-    Then o usuário "acruchi" é encaminhado para uma pagina de esqueci a senha
-    And preenche o campo de cpf
-    And seleciona recuperar a senha 
-    Then o usuário "acruchi" é encaminhado para a pagina de recuperar a senha
-    And o usuário "acruchi" recebe o código
-    When o usuário "acruchi" preenche o campo do código com o código correto
-    Then o usuário "acruchi" recebe uma mensagem "digite uma nova senha"
-    And o usuário "acruchi" digita uma nova senha
-    Then é encaminhado para a pagina de login 
+Scenario: Recuperar a senha correta
+    Given o usuário está na página de "esqueci a senha"
+    When o usuário preenche o "email" corretamente
+    And recebe uma mensagem "Código enviado para o email"
+    Then é encaminhado para a página de "recuperar a senha"
+  
+Scenario: Recuperar a senha incorreta
+    Given o usuário está na página de "esqueci a senha"
+    When o usuário preenche o email "usererror@gmail.com" 
+    Then recebe uma mensagem "Email inválido"
 
-Scenario: Esqueceu a senha e preenche o campo de recuperar com o código errado
-    Given o usuário "acruchi" não está logado  na aplicação
-    And está na página de "login"
-    When o usuário "acruchi" seleciona a opção de Esqueceu a senha
-    Then o usuário "acruchi" é encaminhado para uma pagina de esqueci a senha
-    And preenche o campo de cpf
-    And seleciona recuperar a senha 
-    Then o usuário "acruchi" é encaminhado para a pagina de recuperar a senha
-    And o usuário "acruchi" recebe o código
-    When o usuário "acruchi" preenche o campo do código com o código "3333"
-    Then o usuário "acruchi" recebe uma mensagem "Código errado"
-    And continua na pagina de recuperar a senha 
+Scenario: Mudando a senha correta
+    Given o usuário está na pagina de "recuperar a senha"
+    When o usuário preenche o "código" corretamente
```

Comment:
Since you chose to separate the password recovery scenarios, you lost the information about who is the user trying to recover their password in this scenario. But, you still need to know the user. Therefore, in addition to providing the recovery code, the user must also provide their email. By adopting this approach, you do not have to merge the scenarios again, you can simply add the step of providing the email before the code step.

Reply:
So the password recovery page does not have to get an authentication or something like that.